addons/website_event_sale/tests/test_frontend_buy_tickets.py

```diff
@@ -6,13 +6,13 @@
 from datetime import timedelta
 
 from odoo.addons.base.tests.common import HttpCaseWithUserDemo
+from odoo.addons.website.tools import MockRequest
+from odoo.addons.website_event_sale.controllers.main import WebsiteEventSaleController
 from odoo.addons.website_event_sale.tests.common import TestWebsiteEventSaleCommon
 from odoo.fields import Datetime
 
 
-@odoo.tests.common.tagged('post_install', '-at_install')
-class TestUi(HttpCaseWithUserDemo, TestWebsiteEventSaleCommon):
-
+class TestUiCommon(HttpCaseWithUserDemo, TestWebsiteEventSaleCommon):
```

Comment:
I think the name of the file and classes are not straightforward. I would do as usual with one class per file, with some kind of frontend "namespace/prefix" as necessary.